data/World/Overworld.json

```diff
@@ -1096,6 +1243,11 @@
         "font_color": "Pink",
         "scene": "Kakariko Village",
         "hint": "Kakariko Village",
+        "locations": {
+            "Kak Near Odd Medicine Building Pot 1": "is_child",
+            "Kak Near Odd Medicine Building Pot 2": "is_child",
```

Comment:
Boomerang trick throw can obtain these, but with the tower js trick on it would only skip Sticks. So it's not worth adding.